pkg/jobconfig/files.go

```diff
@@ -414,7 +414,9 @@ func mergePresubmits(old, new *prowconfig.Presubmit) prowconfig.Presubmit {
 	merged.AlwaysRun = old.AlwaysRun
 	merged.RunIfChanged = old.RunIfChanged
 	merged.Optional = old.Optional
-	merged.MaxConcurrency = old.MaxConcurrency
+	if old.MaxConcurrency != 0 {
```

Comment:
Sorry, I don't understand this. We need the setting only for postsubmits and postsubmits must have this set to `1` as we only have build jobs as postsubmits. Why the check?

Reply:
Is this up to date?
https://github.com/openshift/ci-tools/blob/master/GENERATOR.md#hand-edited-prow-configuration

It did mention both presubmits and postsubmits.
However, periodic is not there.

Our prow-gen sets 1 only for the promotion job. We could have other postsubmits, right?

https://github.com/openshift/ci-tools/blob/9f3e703ba3e72ad422e61e105b95de682cb092a1/pkg/prowgen/prowgen.go#L175-L180

Reply:
Right, we could have other postsubmits but I think there is no reason why a postsubmit should ever not run with max_concurrency: 1.

Reply:
It is the case for the current jobs.
This returns nothing.
> grep -irn max_ ./ci-operator/jobs | grep -v presubmits.yaml | grep -v "max_concurrency: 1"

Let me do it.

Reply:
Not doing as discussed over slack.